en/docs/try-out/account-information-service-flow.md

```diff
@@ -0,0 +1,896 @@
+
+18. Use the left menu panel and go to **API Configurations > Endpoints**. 
+
+    ![select_endpoints](../assets/img/get-started/quick-start-guide/select-endpoints.png)
+
+19. Add a **Dynamic Endpoint**. ![add_dynamic_endpoint](../assets/img/get-started/quick-start-guide/add_dynamic_endpoint.png)
+
+20. Go to **Deployments** using the left menu pane. 
+
+    ![select_deployments](../assets/img/get-started/quick-start-guide/select-deployments.png)
+    
+21. Select the API Gateway type, in this scenario, it is **Default**. ![api_gateway](../assets/img/get-started/quick-start-guide/dcr-api-gateway.png)
+
+22. Click **Deploy**.
+
+23. Go to **Overview** using the left menu pane. 
+
+    ![select_overview](../assets/img/get-started/quick-start-guide/select-overview.png)
+
+24. Click **Publish**. ![publish_api](../assets/img/get-started/quick-start-guide/publish-api.png)
+
+## Subscribing to PSD2 API
+
+1. The deployed API is now available in the Developer Portal at `https://<APIM_HOST>:9443/devportal`.
+
+2. Select the **NextGenPsd2XS2AFramework v1** API.
+ 
+3. Locate **Subscriptions** from the left menu pane. 
+
+    ![select_subscriptions](../assets/img/get-started/quick-start-guide/select-subscriptions.png)
+    
+4. From the **Application** dropdown, select the application that you want to be subscribed to the NextGenPsd2XS2AFramework v1 API. ![subscribe_to_api](../assets/img/try-out/account-flow/subscribe-to-api.png)
+
+5. Click **Subscribe**.
+
+## Invoking PSD2 API
+
+### Generating application access token
+
+Once you register the application, generate an application access token.
+
+1. Generate the client assertion by signing the following JSON payload using supported algorithms. 
+
+    !!! note
+        If you have configured the [OB certificates](https://openbanking.atlassian.net/wiki/spaces/DZ/pages/252018873/OB+Root+and+Issuing+Certificates+for+Sandbox), 
+        download the certificate and keys attached [here](../../assets/attachments/Certificates.zip), and use them for signing and transports layer security testing purposes.
+    
+    ``` tab='Format'
+    
+    {
+    "alg": "<The algorithm used for signing.>",
+    "kid": "<The thumbprint of the certificate.>",
+    "typ": "JWT"
+    }
+     
+    {
+    "iss": "<This is the issuer of the token. For example, client ID of your application>",
+    "sub": "<This is the subject identifier of the issuer. For example, client ID of your application>",
+    "exp": <This is the epoch time of the token expiration date/time>,
+    "iat": <This is the epoch time of the token issuance date/time>,
+    "jti": "<This is an incremental unique value>",
+    "aud": "<This is the audience that the ID token is intended for. For example, https://<IS_HOST>:9446/oauth2/token>"
+    }
+     
+    <signature: For DCR, the client assertion is signed by the private key of the signing certificate. For other scenarios, use the private signature of the application certificate.>
```

Comment:
Use the sample provided in the google doc here because there is no DCR for Berlin

Reply:
Fixed in 49adb6d8c05809680c301f250e674df1f369b3f0